### ISSUE TITLE
[Feature Request] get dev builds

### ISSUE THREAD

**Issue by zhigang1992** [shreyas1307/hackintosh-pkg-cli#8]
For us that's on Big Sur, stable build doesn't cut it anymore. 😭 

We need build from 

http://kexts.goldfish64.com/

And

http://drivers.goldfish64.com/

**Comment by shreyas1307** [comment]
Hi, thanks for leaving your feedback. I'll take a look over them and try to include them in the coming weeks.

**Comment by khrj** [comment]
Hi! 
I was wondering how this would be possible? I've tried to build a downloader tool myself earlier, but I got stuck (😭) when I tried to download kexts from one drive, which doesn't offer any static URLs or APIs that I know of.

**Comment by zhigang1992** [comment]
right, another option would be to clone these repos and then build them directly locally. :)

**Comment by khrj** [comment]
That wouldn't be possible on Windows and linux, and even if you're using macOS you'd need Xcode + It would take time to build
